docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolthink.md

```diff
@@ -0,0 +1,26 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Think Tool node documentation
+description: Learn how to use the Think Tool node in n8n. Follow technical documentation to integrate the Tool Think node into your workflows.
+contentType: [integration, reference]
+priority: medium
+---
+
+# Think Tool node
+
+The Think Tool node is a [tool](/glossary.md#ai-tool) that invites an [agent](/glossary.md#ai-agent) to do some thinking before answering.
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/sub-node-expression-resolution.md"
+
+## Templates and examples
+
+<!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
+[[ templatesWidget(page.title, 'wikipedia') ]]
```

Comment:
Is "wikipedia" the right keyword for this one? If there's not currently an [integrations page](https://n8n.io/integrations/) for it, you can use a placeholder like `think-tool`.